tests/www/api/experimental/test_endpoints.py

```diff
@@ -80,239 +90,265 @@ def test_info(self):
         self.assertEqual(version, resp['version'])
 
     def test_task_info(self):
-        url_template = '/api/experimental/dags/{}/tasks/{}'
+        with conf_vars(
+            {("core", "store_serialized_dags"): self.dag_serialzation}
+        ):
+            url_template = '/api/experimental/dags/{}/tasks/{}'
```

Comment:
I think using `conf_vars` as decorator will reduce number of code changes :)

Reply:
This will not work properly with`parameterized_class`